src/main/java/duke/TasksList.java

```diff
@@ -151,6 +151,13 @@ public String importStorageStrings(List<String> tasksStrings) throws InvalidArgu
         return response;
     }
 
+    /**
+     * Finds all tasks that contain a certain keyword.
+     *
+     * @param instruction User input.
+     * @return Tasks in text format that contains keyword to be found.
+     * @throws InvalidArgumentException If there is no keyword inputted.
+     */
     public String find(List<String> instruction) throws InvalidArgumentException {
```

Comment:
This is fine, but I think you could increase readability by changing the method name to "findMatchingTasks" or something along that line.